docs/config.md

```diff
@@ -1,174 +1,39 @@
-
-### disable_remote_config
-
-If this config parameter is set to True in `Default` or `System` configuration,
-the remote configuration (`web_cache.json`) will not be loaded.
+The rest of this section will assume you have shell access to your device.
 
+## How it works
 
-## Meta Configuration
+OVOS will load configuration files from several locations and combine them into a single `json` file that is used throughout the software.
 
-while `mycroft.conf` configures the voice assistant, `ovos.conf` configures the library
-
-what this means is that `ovos.conf` decides what files are loaded by the `Configuration` class described above, as an end user or skill developer you should never have to worry about this
-
-all XDG paths across OpenVoiceOS packages build their paths taking `ovos.conf` into consideration
-
-this feature is what allows downstream voice assistants such as neon-core to change their config files to `neon.yaml`
-
-Using the above example, if `Configuration()` is called from `neon-core`, the following configs would be loaded in this
-order:
-
-- `{core-path}`/configuration/neon.yaml
-- `os.environ.get('MYCROFT_SYSTEM_CONFIG')` or /etc/neon/neon.yaml
-- `os.environ.get('MYCROFT_WEB_CACHE')` or `XDG_CONFIG_PATH`/neon/web_cache.json
-- ~/.neon/neon.yaml (Deprecated)
-- `XDG_CONFIG_DIRS` + /neon/neon.yaml
-- /etc/xdg/neon/neon.yaml
-- `XDG_CONFIG_HOME` (default ~/.config) + /neon/neon.yaml
-
-
-### ovos.conf
-
-The `ovos_config` package determines which config files to load based on `ovos.conf`.
-`get_ovos_config` will return default values that load `mycroft.conf` unless otherwise configured.
-
-`ovos.conf` files are loaded in the following order, with later files taking priority over earlier ones in the list:
-
-- /etc/OpenVoiceOS/ovos.conf
-- /etc/mycroft/ovos.conf (Deprecated)
-- `XDG_CONFIG_DIRS` + /OpenVoiceOS/ovos.conf
-- /etc/xdg/OpenVoiceOS/ovos.conf
-- `XDG_CONFIG_HOME` (default ~/.config) + /OpenVoiceOS/ovos.conf
-
-A simple `ovos_config` should have a structure like:
-
-```json
-{
-  "base_folder": "mycroft",
-  "config_filename": "mycroft.conf",
-  "default_config_path": "<Absolute Path to Installed Core>/configuration/mycroft.conf",
-  "module_overrides": {},
-  "submodule_mappings": {}
-}
-```
+- `{ovos-config-path}/mycroft.conf`
+  - usually in `<python_install_path>/site-packages/ovos_config/mycroft.conf`
+  - This is the default configuration file distrubuted with `ovos-core`
+- `os.environ.get('MYCROFT_SYSTEM_CONFIG')` or /etc/mycroft/mycroft.conf
+  - This is the default configuration file used by images, and may change specific values to corispond with how the image works.
+- `os.environ.get('MYCROFT_WEB_CACHE')` or `XDG_CONFIG_PATH`/mycroft/web_cache.json
+- `~/.mycroft/mycroft.conf` (Deprecated)
+- `XDG_CONFIG_DIRS` + /mycroft/mycroft.conf
+- `/etc/xdg/mycroft/mycroft.conf`
+- `XDG_CONFIG_HOME` (default ~/.config) + /mycroft/mycroft.conf
+  - This is the file that you should use to modify the configuration.
```

Comment:
Consider adding this note at the top of the list, or reversing the order of the list, to avoid confusion with users who read the first couple lines of documentation and end up editing the wrong config file.

Reply:
added a note to the top